e2e/wdio/core/tests/upload-collection.e2e-spec.ts

```diff
@@ -0,0 +1,168 @@
+import { UploadCollectionPo } from '../pages/upload-collection.po';
+import {
+    acceptAlert,
+    click,
+    getAlertText,
+    getAttributeByName,
+    getElementArrayLength,
+    getElementSize,
+    getText,
+    isElementClickable,
+    refreshPage,
+    scrollIntoView,
+    setValue,
+    uploadFile
+} from '../../driver/wdio';
+import {
+    acceptAlertText,
+    declineAlertText,
+    formatArr,
+    imagePath,
+    imageText,
+    testText
+} from '../fixtures/appData/upload-collection-contents';
+import { checkElArrIsClickable } from '../../helper/assertion-helper';
+
+describe('File uploader component test', () => {
+    const uploadCollectionPage = new UploadCollectionPo();
+    const {
+        uploadCollectionComplexExample,
+        uploadCollectionExample,
+        link,
+        editButton,
+        input,
+        okButton,
+        declineButton,
+        item,
+        uploadCollectionSmallExample,
+        uploadCollectionCustomExample,
+        emphasizedButton,
+        fileUploaderInputFile,
+        checkbox,
+        standardButton
+    } = uploadCollectionPage;
+
+    beforeAll(() => {
+        uploadCollectionPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+    }, 2);
+
+    describe('Should check default example', () => {
+        it('should verify basic checks', () => {
```

Comment:
you check here 3 different checks, would be better to divide them into 3 iterations to easily debug if something fails. Same for other iterations `verify basic checks`, please